src/d3d11/d3d11_annotation.cpp

```diff
@@ -1,9 +1,12 @@
 #include "d3d11_annotation.h"
+#include "d3d11_context.h"
+#include "d3d11_device.h"
 
 namespace dxvk {
 
   D3D11UserDefinedAnnotation::D3D11UserDefinedAnnotation(ID3D11DeviceContext* ctx)
-  : m_container(ctx) { }
+  : m_container(ctx),
```

Comment:
Now that we actually interact with the implementation, `ctx` should be changed to `D3D11DeviceContext` to avoid the useless casts.

Reply:
Sure.